src/features/compendium/components/SidebarArrayView.vue

```diff
@@ -60,6 +60,11 @@ export default Vue.extend({
       required: false,
       default: '',
     },
+    subConditional: {
+      type: String,
+      required: false,
+      default: '',
+    },
```

Comment:
This is a little strange, but it seemed the best way to make this generic.
Wait, this may be just broken for non-background items now.

Reply:
Fixed with the recent change on line 28.